.codecov.yml

```diff
@@ -28,11 +28,11 @@ coverage:
         branches:
           - master
         if_no_uploads: error
-        if_not_found: success
+        if_not_found: error
         if_ci_failed: error
         only_pulls: false
 
 # Files to ignore
 ignore:
   - "data"
-  - "coverage"
+  - "notebooks"
```

Comment:
We will have to test the notebooks actually, but we can leave this for later I guess.